opentelemetry-semantic-conventions/scripts/generate-consts-from-spec.sh

```diff
@@ -21,10 +21,10 @@ md_to_rs() {
 		# Find markdown tables (lines starting with "|") and take the first 2 columns
 		# - The 1st column is the attribute name consisting of a-z0-9._
 		# - The 2nd column is the description
-		if [[ "$line" =~ ^\|\ \`?([a-z0-9._]+)\`?\ +\|\ ([^|]+) ]]; then
+		if [[ "$line" =~ ^\|\ \[?\`?([a-z0-9._]+)\`?\]?(:?\([^|]+\))?\ +\|\ ([^|]+)\ +\|\ ([^|]+) ]]; then
```

Comment:
I think we should either update the comment about the regex in the line above or remove it, if it's not helpful. 🙂

Reply:
Yeah good idea